svg/global_attributes.json

```diff
@@ -43,12 +43,12 @@
           "spec_url": "https://svgwg.org/svg2-draft/struct.html#LangSpaceAttrs",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": false
```

Comment:
https://mdn-bcd-collector.gooborg.com/tests/svg/global_attributes/lang looks to me like an invalid test for this. It's looking for a `lang` property, but there is no `lang` IDL attribute on `SVGElement` or any subinterface.